daemon/prune.go

```diff
@@ -25,6 +25,27 @@ var (
 	// ErrPruneRunning is returned when a prune request is received while
 	// one is in progress
 	ErrPruneRunning = fmt.Errorf("a prune operation is already running")
+
+	containersAcceptedFilters = map[string]bool{
+		"label":  true,
+		"label!": true,
```

Comment:
I don't think the later is a valid syntax for filters (e.g. `filters="label != foo.bar"`)